src/apis/Dimensions/index.js

```diff
@@ -44,6 +46,26 @@ export default class Dimensions {
       scale: win.devicePixelRatio || 1,
       width: win.screen.width
     };
+
+    if (canUseDOM) {
+      const change: window.Event = new window.Event(EVENT_CHANGE);
```

Comment:
Why did you choose to use a custom DOM event? Browser support doesn't appear to be there either